packages/core/strapi/lib/core/loaders/plugins/get-enabled-plugins.js

```diff
@@ -106,6 +103,7 @@ const getEnabledPlugins = async strapi => {
 
   const enabledPlugins = pipe(
     defaultsDeep(declaredPlugins),
+    pickBy((pValue, pName) => _.get(declaredPlugins, [pName, 'enabled'], true)),
```

Comment:
is this necessary? the last pickBy in the list should be doing the job

Reply:
No It's not, I just pushed commit